appveyor.yml

```diff
@@ -14,6 +14,7 @@ install:
 
 build_script:
   - travis-tool.sh install_deps
+  - travis-tool.sh install_r Rcpp
```

Comment:
Rcpp is not being installed probably due to https://github.com/hadley/devtools/issues/1246 or possibly https://github.com/hadley/devtools/issues/1409. Use devel devtools in appveyor should fix it.